Source/CLI/rawcooked.1 [english edits]

```diff
@@ -0,0 +1,223 @@
+.br
+If the input is a Matroska container with computed hash files stored in an attachment, then decode the Matroska input and check hashes. Any issue raised by the check is considered as invalid data.
+.br
+This is default when the input is a Matroska container and RAWcooked metadata contains hashes, and when the input is a Matroska container with hash files in the attachment. This default may change in the future.
+.TP
+.B \-\-no\-hash
+Do not compute or test the hash of the file (see above).
+.br
+This is default when the input is RAW content, but may change in the future.
+.SS INPUT RELATED OPTIONS
+.TP
+.B \-\-file
+Unlock the compression of files, for example with .dpx or .wav.
+.TP
+.B \-framerate \fIvalue
+Force the video frame rate value to \fIvalue\fR.
+.br
+Default frame rate value is found in the image file metadata, if available. Otherwise it will default to \fI24\fR.
+.SS ENCODING RELATED OPTIONS
+.TP
+.B \-c:a \fIvalue
+Use this command to force the audio encoding format to \fIvalue\fR: \fIcopy\fR (for example copy PCM to PCM, without modification), \fIFLAC\fR
+.br
+The default value is \fIFLAC\fR.
+.TP
+.B \-c:v \fIvalue
+Force the video encoding format \fIvalue\fR: only \fIFFV1\fR is currently allowed.
+.br
+The default value is \fIFFV1\fR.
+.TP
+.B \-coder \fIvalue
+If video encoding format is \fIFFV1\fR, set the Coder to \fIvalue\fR: \fI0\fR (Golomb-Rice), \fI1\fR (Range Coder), \fI2\fR (Range Coder with custom state transition table).
+.br
+The default value is \fI1\fR.
+.TP
+.B \-context \fIvalue
+If the video encoding format is \fIFFV1\fR, set the Context to \fIvalue\fR: \fI0\fR (small), \fI1\fR (large).
+.br
+The default value is \fI0\fR.
+.TP
+.B \-format \fIvalue
+Set the container format to \fIvalue\fR: only \fIMatroska\fR is currently allowed.
+.br
+The default value is \fIMatroska\fR.
+.TP
+.B \-g \fIvalue
+If video encoding format is \fIFFV1\fR, set the GOP size to \fIvalue\fR\: \fI1\fR (generates a strict intra-frame bitstream), \fI0\fR (allows adaptable context model across frames).
+.br
+The default value is \fI1\fR. Ensure you leave the setting at \fI1\fR for archival use.
+.TP
+.B \-level \fIvalue
+The video encoding format \fIFFV1\fR can have Version set to \fIvalue\fR: \fI0\fR, \fI1\fR, \fI3\fR.
+.br
+The default value is the latest version \fI3\fR.
+.TP
+.B \-slicecrc \fIvalue
+If video encoding format is \fIFFV1\fR, you can set the CRC checksum to \fIvalue\fR: \fI0\fR (CRC checksums off), \fI1\fR (CRC checksum on).
+.br
+The default value is \fI1\fR.
+.TP
+.B \-slices \fIvalue
+If the video encoding format is \fIFFV1\fR, you can set the multithreaded encoding slices to \fIvalue\fR: any integer over 1 (it is recommended to use a figure divisible by your workstations CPU core processors such as 2, 4, 6, 9, 16, 24…).
+.br
+The default value is variable between \fI16\fR and \fI512\fR, depending on the video frame size and depth.
+.SH COPYRIGHT
+Copyright (c) 2019 MediaArea.net SARL & AV Preservation by reto.ch
```

Comment:
Why is 2018 deleted?

Reply:
Whoops. Thought it was out of date! Ignore me :)